app/assets/stylesheets/typography.scss

```diff
@@ -0,0 +1,41 @@
+h1, h2, h3, h4, p {
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+h1 {
+  font-size: 36px;
```

Comment:
duh of course.

don't know why I didn't do it to begin with. (and `rem` is much better in this case 👍